### DISCUSSION TITLE
Welcome to SimSiam Discussions!

### DISCUSSION THREAD

**Discussion by PatrickHua** [PatrickHua/SimSiam#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qianandfei** [comment]
Some mistakes exist this code. such as , a undefined name ‘np’ , ‘Image’ ,  'T',   'GaussianBlur' and the like， exist in running.  In addtion,  another warning that module 'torchvision.transforms' has no attribute 'GaussianBlur' happen to me.

**Comment by qianandfei** [comment]
Besides， what's means？ in line 71 of main.py ---"global_progress = tqdm(range(0, args.stop_at_epoch), desc=f'Training'  ,"desc=f'Epoch {epoch}/{args.num_epochs}'" .   Obviously ,  it is invalid syntax.
